### DISCUSSION TITLE
Promotion page for CleanRL-based projects

### DISCUSSION THREAD

**Discussion by cpwan** [vwxyzjn/cleanrl#363] [Ideas]
Sharing CleanRL-based projects can be a great way to broaden the CleanRL community and foster collaborations among individuals from diverse backgrounds, such as RL researchers, deep learning researchers, and domain experts.

For instance, my team has successfully applied CleanRL to solve operation research problems with a large margin of speedup. We recently released [RLOR: A Flexible Framework of Deep Reinforcement Learning for Operation Research](https://github.com/cpwan/RLOR). We are looking for collaboration from diverse backgrounds.

As the CleanRL community grows, it would be great if we have a channel for the community to get involved in interesting new projects 😄

**Comment by vwxyzjn** [comment]
Hi @cpwan, thanks for sharing your work! Super cool to see RL being used in operation research. Right now, we just have a bare-bone version of community projects showcase at https://docs.cleanrl.dev/cleanrl-supported-papers-projects/, but would be happy to find ways to showcase people's works better :)
